blog/_posts/2017-11-30-bloop-release.md

```diff
@@ -0,0 +1,158 @@
+
+As of this announcement, `bloop` is capable of compiling and testing your project, and it integrates
+with `sbt`. It supports all the test frameworks sbt supports (`JUnit`, `Scalatest`, `Scalacheck`,
+`utest` and more).
+
+`bloop` is a young project with only three weeks of life. That means that
+there's a lot of work left before we support all different platforms and needs.
+In our next release, we want to focus on:
+
+- Stabilizing the internals of the project.
+- Supporting `console`, [a faster `testQuick`](https://github.com/scalacenter/bloop/issues/61), and
+  [built-in file watching](https://github.com/scalacenter/bloop/issues/7).
+- Making more performance improvements to our integration, to be even faster.
+* Exploring integrations with other build tools like `Maven` and `Gradle`.
+- Supporting Dotty and out-of-the-box cross-compilation to
+  [Scala.js](https://github.com/scala-js/scala-js/) and [Scala
+  Native](https://github.com/scala-native/scala-native).
+
+### First beta release `0.1`
+
+The first beta version of bloop can be used following the instructions in [scalacenter/bloop][].
+
+### Preserving hot compilers
+
+An important feature of `bloop` is its [nailgun][] integration. Nailgun is a client, protocol, and
+server for running Java programs from the command line without incurring the JVM startup overhead.
+With it, `bloop` ensures that you always have a hot compiler in the background to compile your
+project. This brings significant improvements indeed: [a hot compiler can compile up to 18 times
+faster than a hot
+compiler](https://scala-ci.typesafe.com/grafana/dashboard/db/scala-benchmark?var-branch=2.12.x&var-source=vector&var-bench=HotScalacBenchmark.compile&var-host=scalabench@scalabench@).
+
+Keeping a hot compiler in the background is crucial if one takes compilation speed and productivity
+seriously. This is often difficult and takes space of mind. For example, `sbt` throws away a hot
+compiler every time users quit (`Ctrl-C`) and restart, but also when they do `reload` after several
+executions of `compile`. By this rule, the more users modify their sbt build or
+have bad habits, the less productive they are.
+
+Current sbt users can indeed avoid quitting and prefer suspending (`Ctrl-Z`),
+but they will always be punished for modifying their builds. In our experience,
+asking users to change their habits is hard, so we have preferred to add
+a built-in nailgun integration in `bloop` that addresses this problem at its
+root rather than changing people's habits, which is hard and time consuming.
+
+### How does it work?
+
+When `bloop` starts up, it loads a build from a configuration directory passed in by the user, which
+defaults to `.bloop-config`. This directory contains a configuration file per every project, and
+every configuration file contains basic information about the project: name, base directory,
+classpath, source directories, classes directory, Scala and Java options and a few more.
```

Comment:
Still unresolved comment from before. It's not clear to the reader why you're sharing this information about the configuration. How does this improve anything? Without explicitly stating what the benefit is of this, a novice user doesn't know why you're sharing these details. It just reads as random extraneous details. Perhaps state why having stuff like classpath and source directories specified ahead of time, etc, is a good, fast thing?

Reply:
I think I explain it badly, I don't mean it's a good thing, I just want to explain how bloop works so that people know that all the changes in their build that changes this field requires a re-generation of their bloop configuration files. And also because I want to educate them.

Reply:
I'm trying to improve this paragraph to make this clearer, I don't do a good job now.

Reply:
Can you read all the section again and tell me if the contextual information I've added makes it easier to read? @heathermiller